scala-package/core/pom.xml

```diff
@@ -155,9 +155,9 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>commons-io</groupId>
-      <artifactId>commons-io</artifactId>
-      <version>2.1</version>
+      <groupId>commons-codec</groupId>
```

Comment:
Why we need "commons-codec" here?

Reply:
The apidoc-generation task fails without it. I think it may be to do with how the classpath is setup for that task but I'm not sure.

Reply:
With the change to remove the provided scope at the top level and then explicitly remove the JARs, this has been resolved.